content/en/docs/concepts/cluster-administration/logging.md

```diff
@@ -133,7 +133,10 @@ Because the logging agent must run on every node, it's common to implement it as
 
 Using a node-level logging agent is the most common and encouraged approach for a Kubernetes cluster, because it creates only one agent per node, and it doesn't require any changes to the applications running on the node. However, node-level logging _only works for applications' standard output and standard error_.
```

Comment:
Is this still true?

To me, the two options are:
- containers write stdout and stderr, but with no agreed format. A node-level agent collects these logs and forwards them for aggregation
- applications in containers send logs, possibly via a sidecar, to an aggregation service. The aggregator could be running inside the cluster, or external to it.
  -  node-level logging can happen, separately, to report things such as system upgrades, hardware failure events, etc


both options have their fans. I'm not convinced the argument is settled either way, nor that it needs to be.

Reply:
I have not looked at this page in detail. I originally wanted feedback on deleting the task pages.
If that is the right thing to do, then go ahead and clean up the associated references and pages.
Let me see if I can update a bit more.